Gems/Profiler/Code/Source/CpuProfiler.cpp

```diff
@@ -350,7 +351,7 @@ namespace Profiler
             // Add the cached regions to the map
             for (auto& cachedTimeRegion : m_cachedTimeRegions)
             {
-                const AZStd::string regionName = cachedTimeRegion.m_groupRegionName.m_regionName;
+                const AZStd::string regionName = cachedTimeRegion.m_groupRegionName.m_regionName.GetCStr();
```

Comment:
This should use GetStringView instead of GetCStr to avoid an unnecessary strlen call on a c-string.
```suggestion
                const AZStd::string regionName = cachedTimeRegion.m_groupRegionName.m_regionName.GetStringView();
```